spec/03-plugins/statsd/log_spec.lua

```diff
@@ -0,0 +1,246 @@
+local helpers = require "spec.helpers"
+local UDP_PORT = 20000
+
+
+describe("Statsd Plugin", function()
```

Comment:
Rename to "Plugin: statsd" (as other plugins, this unifies the output from busted in travis and local tests across all plugins).